infra/tpu-pytorch-releases/artifacts.auto.tfvars

```diff
@@ -56,6 +56,22 @@ versioned_builds = [
     accelerator     = "tpu"
     bundle_libtpu   = "0"
```

Comment:
Consider adding python_version = "3.8" explicitly, to make it clear why this build is different from other 2.1 builds

Reply:
Thanks, updated